```
StackOverflowError()
```

L'appel de fonction a dépassé la taille de la pile d'appels. Cela se produit généralement lorsqu'un appel se récursive indéfiniment.
